script.js

```diff
@@ -1,69 +1,82 @@
+const titleInput = document.getElementById('book');
+const authorInput = document.getElementById('author');
 const form = document.getElementById('form');
-const inputBook = document.getElementById('book');
-const inputAuthor = document.getElementById('author');
-const Container = document.getElementById('container');
-const storedBookLocal = [];
+const container = document.getElementById('container');
+const bookskey = 'books';
+let deleteBtns = document.querySelectorAll('#remove');
 
-// Add Books
-
-const addBook = () => {
-  form.addEventListener('submit', (e) => {
-    e.preventDefault();
-
-    const authorbook = inputAuthor.value;
-    const titlebook = inputBook.value;
-    const itemlist = document.createElement('div');
-    itemlist.className = 'Itemlist';
-
-    const obj = {
-      Title: titlebook,
-      Author: authorbook,
-    };
+class Book {
+  constructor(title, author, id) {
+    this.author = author;
+    this.title = title;
+    if (id) {
+      this.id = id;
+    } else {
+      this.id = 1;
+    }
+  }
 
-    const book = `
-    <ul class=''>
-    <li>${obj.Title}</li>
-    <li>${obj.Author}</li>
-    </ul>
+  AddBook() {
```

Comment:
- I  tried to add book but not response when the  add book submit button is clicked. Please fix this.

Reply:
Hi Bobb,

I have try to clone again my project but I see that every thing is working perfectly.

Cheer

Kagheni Mbale Christian

Reply:
The same issue persists even now.